asreview/webapp/api.py

```diff
@@ -389,11 +389,11 @@ def api_get_project_data(project_id):  # noqa: F401
         statistics["filename"] = filename
 
     except FileNotFoundError as err:
-        print(err)
+        logging.info(err)
```

Comment:
Should be logging.error probably?

Reply:
Thanks. This one is oke. No file found is a 200 response.